src/lib/expansion/expansion.md

```diff
@@ -99,9 +99,26 @@ panel can be expanded at a given time:
 </mat-accordion>
 ```
 
+### Lazy rendering
+
+If you want some content not to be rendered until its expansion panel is opened, you can wrap it in
+an `ng-template` using the `matExpansionPanelContent` directive.
```

Comment:
How about
`````markdown
By default, the expansion panel content will be initialized even when the panel is closed.
To instead defer initialization until the panel is open, the content should be provided as
an `ng-template`:
```html
<mat-expansion-panel>
  <mat-expansion-panel-header>
    This is the expansion title
  </mat-expansion-panel-header>

  <ng-template matExpansionPanelContent>
    Some deferred content
  </ng-template>
</mat-expansion-panel>
```
`````